admin/server/organizations.go

```diff
@@ -74,6 +75,36 @@ func (s *Server) GetOrganization(ctx context.Context, req *adminv1.GetOrganizati
 		perms.ReadProjects = true
 	}
 
+	if org.CachedPlanDisplayName == nil {
+		sub, err := s.admin.Biller.GetActiveSubscription(ctx, org.BillingCustomerID)
```

Comment:
Maybe add a comment that it's for backwards compatibility? Hopefully we can eventually remove it.

Reply:
There can be some delay in receiving or handling webhook request so its not just for backwards compatibility

Reply:
I think in practice it's better to return the plan name as unpopulated or `"Unknown plan"` than try to call the Orb API in that case. The `GetOrganization` API call is important for every page load and many CLI commands, which would create a dangerous dependency if Orb is unavailable.

Reply:
Makes sense but for now we keep it and remove in later versions ? Any trick to not use it for new orgs but only for older orgs. May be we run a one time job to populate display name or triggerable cmd that syncs plan names with org and we manually run it once and remove this code ?